lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+  },
+  "flashing-micropython": {
+    "defaultMessage": "Transferint MicroPython ",
+    "description": "Progress dialog text for full flashes (slower)"
+  },
+  "font-size": {
+    "defaultMessage": "Mida de la font",
+    "description": "Font size option text"
+  },
+  "guide-link": {
+    "defaultMessage": "Per obtenir més suport, consulta la guia de l'editor de Python <link>Python Editor guide </link>",
+    "description": "Text with an external link to support/help content"
+  },
+  "help": {
+    "defaultMessage": "Ajuda",
+    "description": "Help menu label"
+  },
+  "help-support": {
+    "defaultMessage": "Ajuda i suport",
+    "description": "Menu item for link to support site"
+  },
+  "help-translate": {
+    "defaultMessage": "Ajuda a traduir",
+    "description": "Help translate  menu option text"
+  },
+  "highlight-code-structure": {
+    "defaultMessage": "Ressaltar l'estructura del programa",
+    "description": "Code structure setting label"
+  },
+  "highlight-code-structure-full": {
+    "defaultMessage": "Ple (full)",
+    "description": "Highlight code structure option"
+  },
+  "highlight-code-structure-none": {
+    "defaultMessage": "Cap (None)",
+    "description": "Highlight code structure option"
+  },
+  "highlight-code-structure-simple": {
+    "defaultMessage": "Simple",
+    "description": "Highlight code structure option"
+  },
+  "ideas-tab": {
+    "defaultMessage": "Idees",
+    "description": "Ideas tab button text. Tab shows programs to give students ideas."
+  },
+  "ideas-tab-description": {
+    "defaultMessage": "Prova aquests projectes, modifica'ls i inspira't",
+    "description": "Ideas tab description text. Tab shows programs to give students ideas."
+  },
+  "insert-code-action": {
+    "defaultMessage": "Insereix codi"
+  },
+  "language": {
+    "defaultMessage": "Idioma",
+    "description": "Language option text"
+  },
+  "less-action": {
+    "defaultMessage": "Menys",
+    "description": "Less button text (showing less content)"
+  },
+  "load-error-makecode-info": {
+    "defaultMessage": "Aquest fitxer hexadecimal no es pot carregar a l'Editor de Python. L'Editor Python no pot obrir fitxers hexadecimals creats amb Microsoft MakeCode.",
+    "description": "Load error message"
+  },
+  "load-error-makecode-link": {
+    "defaultMessage": "Visita <link>https://makecode.microbit.org/</link> per utilitzar aquest fitxer hexadecimal.",
+    "description": "Load error message"
+  },
+  "load-error-mixed": {
+    "defaultMessage": "Un fitxer hexadecimal només es pot carregar sol. Substitueix tots els fitxers del projecte.",
+    "description": "Load error message"
+  },
+  "load-error-mpy": {
+    "defaultMessage": "Aquesta versió de l'Editor de Python actualment no admet l'addició de fitxers .mpy.",
+    "description": "Load error message"
+  },
+  "load-error-title": {
+    "defaultMessage": "No es pot carregar {fileCount, plural, one {file} other {files}}",
```

Comment:
ICU needs review.
Maybe correct but check "file" and "files" and pluralisation.

Reply:
Requested a review of all ICU strings in Crowdin